modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/handle/migrate/MigrateDeployProcess.java

```diff
@@ -35,12 +38,12 @@ public class MigrateDeployProcess implements Serializable, JavaDelegate {
 
     @Autowired
     public void setRuntimeService(RuntimeService runtimeService) {
-        this.runtimeService = runtimeService;
+        MigrateDeployProcess.runtimeService = runtimeService;
```

Comment:
@baixinsui I believe it better to keep all autowired beans as non-static. I had actually given the same comment when the original PR was opened as well. 
Could you please make the change to MigrateDeployProcess and MigrateDestroyProcess?

Reply:
@swaroopar The workflow engine couldn't instantiate class after I change the static class to autowired bean.
![image](https://github.com/eclipse-xpanse/xpanse/assets/121531362/03613c92-cdc4-4753-b159-5f4f50fc3c54)